programmer-skills/start-to-develop-js.md

```diff
@@ -239,6 +238,6 @@ and bad practices. The two most popular are:
 - [jslint](http://www.jslint.com/)
 - [jshint](http://jshint.com/)
 
-Both of them have online tools for check your code, but there are
+Both of them have online tools for checking your code, but there are
```

Comment:
"but there are also plugins for almost every code editor, as well as for the command line"